--- a/src/mediaenc_ffmpeg.cpp.orig	2014-10-20 02:43:34.000000000 -0300
+++ b/src/mediaenc_ffmpeg.cpp	2014-12-23 12:06:22.245121159 -0300
@@ -140,7 +140,7 @@ bool wxFfmpegMediaEncoder::BeginEncode(c
 	// write the stream header
 	AVDictionary *opts = NULL;
 	av_dict_set(&opts, "packet_size", "2048", 0);
-	av_dict_set(&opts, "muxrate", "10080000", 0);
+	av_dict_set(&opts, "muxrate", "4100000", 0);
 	char tmpstr[48];
 	snprintf(tmpstr, sizeof(tmpstr), "%i", (int)(0.5 * AV_TIME_BASE));
 	av_dict_set(&opts, "preload", tmpstr, 0);
